PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -122,8 +151,14 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         mvaID = cms.InputTag("photonMVAValueMapProducer:PhotonMVAEstimatorRunIIFall17v2Values"),
         PFIsoChg = cms.InputTag("isoForPho:PFIsoChg"),
         PFIsoAll = cms.InputTag("isoForPho:PFIsoAll"),
+        PFIsoChgQuadratic = cms.InputTag("isoForPho:PFIsoChgQuadratic"),
+        PFIsoAllQuadratic = cms.InputTag("isoForPho:PFIsoAllQuadratic"),
+        HoverEQuadratic = cms.InputTag("hOverEForPho:HoEForPhoEACorr"),
```

Comment:
These 3 user floats are not used anywhere, are they? Don't you want to add them to the photon table?

Reply:
Yes, I will add them there.